nixos/tests/kanidm.nix

```diff
@@ -44,11 +44,25 @@ import ./make-test-python.nix ({ pkgs, ... }:
         enableClient = true;
         clientSettings = {
           uri = "https://${serverDomain}";
+          verify_ca = true;
+          verify_hostnames = true;
+        };
+        enablePam = true;
+        unixSettings = {
+          pam_allowed_login_groups = [ "shell" ];
+          default_shell = "/run/current-system/sw/bin/bash";
+          home_prefix = "/home/";
+          home_attr = "uuid";
+          home_alias = "spn";
+          uid_attr_map = "spn";
+          gid_attr_map = "spn";
```

Comment:
Are all of these needed or are they the defaults?

Reply:
I checked again and only `pam_allowed_login_groups` is required.